nuxt.config.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  server: {
+    host: '0.0.0.0' // default: localhost
```

Comment:
I'm not sure I understand the comment. Is this something people need to change to run locally?

Reply:
Oh, no it was just the default comment that the nuxtjs docs included. I'll remove it.